3ds Max/Max2Babylon/GlobalUtility.cs

```diff
@@ -260,10 +261,15 @@ private void InstallMenus()
             menu.AddItem(menuItemBabylon, -1);
 
             menuItemBabylon = Loader.Global.IMenuItem;
-            menuItemBabylon.Title = "Babylon Actions Builder";
+            menuItemBabylon.Title = "Babylon Invert Skip Flatten Status";
```

Comment:
Can we use a better menu name here as well?

Reply:
of course, is now "Babylon Toggle Skip Flatten Status"